modules/virt-pausing-vm-web.adoc

```diff
@@ -19,15 +19,15 @@ You can pause a virtual machine (VM) from the web console.
 * To stay on this page, where you can perform actions on multiple VMs:
 
 .. Click the Options menu {kebab} located at the far right end of the row and click *Pause VirtualMachine*.
+.. If action confirmation is enabled, click *Pause* in the confirmation dialog.
 
 * To pause the VM from the tree view:
-
 .. Click the *>* icon next to the project name to open the list of VMs.
-
 .. Right-click the name of the VM and select *Pause*.
-
+.. If action confirmation is enabled, click *Pause* in the confirmation dialog.
 * To view comprehensive information about the selected VM before you pause it:
 
 .. Access the *VirtualMachine details* page by clicking the name of the VM.
 
 .. Click *Actions* -> *Pause*.
+.. If action confirmation is enabled, click *Pause* in the confirmation dialog.
```

Comment:
Suggestions in this file to illustrate what I am thinking, but would apply across the PR.
```suggestion
.. If actions confirmation is enabled, click *Pause* in the confirmation dialog.
```